### ISSUE TITLE
Villagers should be able to sleep through the day

### ISSUE THREAD

**Issue by KingMushroomBoy** [Ladysnake/Requiem#541]
How about you make villagers when you possess them allowed to sleep in day? They can in the real one.

**Comment by KingMushroomBoy** [comment]
I had a question about quilt BUT I didn't really care so the whole comment changed.

**Comment by Pyrofab** [comment]
Huh, I guess they can. Well, maybe - so if you sleep as a villager at night it goes to day and during the day it goes to night ?

**Comment by KingMushroomBoy** [comment]
> Huh, I guess they can. Well, maybe - so if you sleep as a villager at night it goes to day and during the day it goes to night ?

Yes. Villagers can also sleep in the nether or end too so make that with the update. Thanks.
